source/Octopus.Tentacle/Security/Certificates/CertificateEncoder.cs

```diff
@@ -103,7 +103,7 @@ static X509Certificate2 FromBase64String(string? thumbprint, string certificateS
             }
         }
         
-#if NET472_OR_GREATER || NETCOREAPP || NETSTANDARD
+#if NETCOREAPP || NETSTANDARD
```

Comment:
This used to be bypassed as we were targeting net452, but breaks net48. With `EphemeralKeySet` the cert loses the private key once added to the key store. As per the doc
> Since the keys are not persisted to disk, certificates loaded with this flag are not good candidates to add to an X509Store.

Reply:
This essentially allows us to keep the same behaviour as `net452`